src/components/rangeInput/RangeInput.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-slider
     ref="rangeInput"
-    v-model="currentValue"
+    v-model="value"
```

Comment:
if you check the history, this is the way it previously was. But if you run a linter on this, it will give you an error saying that mutations are not allowed for a property. We need to fix this in a way it doesn't use the actual property. Check this for more context
https://michaelnthiessen.com/avoid-mutating-prop-directly/